recipes/sphinxcontrib-details-directive/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "sphinxcontrib-details-directive" %}
+{% set version = "0.1.0" %}
+{% set hash = "78bd6a67f786a21868abf0e6a5973340d7e7a6fd71b1890de9c856f92877b38b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - sphinx >=2.0
+
+test:
+  imports:
+    - sphinxcontrib  
+    - sphinxcontrib.details
+    - sphinxcontrib.details.directive
+
+about: 
+  home: https://github.com/tk0miya/sphinxcontrib-details-directive
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: 'A sphinx extension adding the details directive'
+  description: |
+    It enables details directive as an element to represent <details> element 
+    in HTML output. It will be converted into mere paragraphs in other output 
+    formats.
+  dev_url: https://github.com/tk0miya/sphinxcontrib-details-directive
+
+extra:
+  recipe-maintainers:
+    - astamminger
+    - chrisjsewell
```

Comment:
@ chrisjsewell are you okay with being a maintainer on this recipe?

Reply:
Hi @mariusvniekerk

consent can be found here: https://github.com/conda-forge/aiida-core-feedstock/issues/12#issuecomment-779866063

Thanks! :)